_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_vaapi.cpp

```diff
@@ -1055,6 +1058,8 @@ mfxStatus VAAPIEncoder::CreateAuxilliaryDevice(
     m_caps.ddi_caps.IntraRefreshBlockUnitSize = 2;
     m_caps.ddi_caps.TileSupport = (attrs[idx_map[VAConfigAttribEncTileSupport]].value == 1);
 
+    m_MaxContextPriority = attrs[ idx_map[VAConfigAttribContextPriority] ].value;
```

Comment:
This attribute might not be supported by the driver. You need to write this:
```
if (attrs[ idx_map[VAConfigAttribContextPriority] ].value != VA_ATTRIB_NOT_SUPPORTED) {
    m_MaxContextPriority = attrs[ idx_map[VAConfigAttribContextPriority] ].value;
}
```

Reply:
Done.